Dockerfile

```diff
@@ -110,6 +114,16 @@ account    sufficient    pam_permit.so\n\
 session    sufficient    pam_permit.so\n\
 ' > /etc/pam.d/sudo
 
+# Install TexLive and required components for Crucible documentation
+RUN mkdir /root/tl && wget https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz -O /dev/stdout |tar -C /root/tl --strip-components=1 -zx  && \
```

Comment:
Can you make sure this link is in the wayback machine in case it goes down in the future.

Reply:
They do a lot of weird redirection with their mirrors.  However, this link is their official way of doing it.

See: https://tug.org/texlive/acquire-netinstall.html

This tarball is only a few megabytes and is just the installer, the actual packages are downloaded by it.

They have a long list of mirrors as well...